e2e/support/helpers/component-testing-sdk/component-embedding-sdk-helpers.tsx

```diff
@@ -21,45 +19,6 @@ export const DEFAULT_SDK_AUTH_PROVIDER_CONFIG = {
   metabaseInstanceUrl: METABASE_INSTANCE_URL,
 };
 
-export const mockAuthProviderAndJwtSignIn = (user = USERS.admin) => {
```

Comment:
We have to move this to a separate module so it does not import `"@metabase/embedding-sdk-react"` into the main app's e2e (which is used by new iframe embedding). In master we've already done this, but in 55 branch that change is not backported.